packages/react-router/src/runtime.ts

```diff
@@ -1,16 +1,18 @@
-import { AbstractRuntime } from "@squide/core";
+import { AbstractRuntime, RootMenuId, type RegisterNavigationItemsOptions, type RegisterRoutesOptions } from "@squide/core";
 import { NavigationItemRegistry, type RootNavigationItem } from "./navigationItemRegistry.ts";
-import { RouteRegistry, type RootRoute } from "./routeRegistry.ts";
+import { RouteRegistry, type RootRoute, type Route } from "./routeRegistry.ts";
 
-export class Runtime extends AbstractRuntime<RootRoute, RootNavigationItem> {
+export class Runtime extends AbstractRuntime<RootRoute | Route, RootNavigationItem> {
     readonly #routeRegistry = new RouteRegistry();
     readonly #navigationItemRegistry = new NavigationItemRegistry();
 
-    registerRoutes(routes: RootRoute[]) {
-        this.#routeRegistry.add(routes);
+    registerRoutes(routes: RootRoute[] | Route[], options: RegisterRoutesOptions = {}) {
+        this.#routeRegistry.add(routes, options);
+
+        const parentLog = options.layoutPath ? `as children of the "${options.layoutPath}" route` : "";
```

Comment:
```suggestion
        const parentLog = options.layoutPath ? ` as children of the "${options.layoutPath}" route` : "";
```
Was missing a space for the next string concatenation.